frontend/src/metabase/collections/components/BulkActions.jsx

```diff
@@ -67,27 +33,66 @@ function BulkActions(props) {
     isNavbarOpen,
   } = props;
 
-  const canMove = selected.every(item => canMoveItem(item, collection));
-  const canArchive = selected.every(item => canArchiveItem(item, collection));
+  const onMoveClick = selected.every(item => canMoveItem(item, collection))
+    ? onMoveStart
+    : null;
+  const onArchiveClick = selected.every(item =>
+    canArchiveItem(item, collection),
+  )
+    ? onArchive
+    : null;
+
+  const showing = selected.length > 0;
 
   return (
-    <BulkActionBar showing={selected.length > 0} isNavbarOpen={isNavbarOpen}>
+    <>
       {/* NOTE: these padding and grid sizes must be carefully matched
-                   to the main content above to ensure the bulk checkbox lines up */}
-      <ActionBarContent>
-        <SelectionControls {...props} />
-        <BulkActionControls
-          onArchive={canArchive ? onArchive : null}
-          onMove={canMove ? onMoveStart : null}
-        />
-        <ActionBarText>
-          {ngettext(
-            msgid`${selected.length} item selected`,
-            `${selected.length} items selected`,
-            selected.length,
-          )}
-        </ActionBarText>
-      </ActionBarContent>
+      to the main content above to ensure the bulk checkbox lines up */}
+      <Motion
+        defaultStyle={{
+          opacity: 0,
+          translateY: 100,
+        }}
+        style={{
+          opacity: showing ? spring(1) : spring(0),
+          translateY: showing ? spring(0) : spring(100),
+        }}
+      >
+        {({ translateY }) => (
+          <BulkActionsToast
+            style={{
+              transform: `translate(-50%, ${translateY}px)`,
+              marginLeft: isNavbarOpen ? NAV_SIDEBAR_WIDTH_HALF : 0,
+            }}
+          >
```

Comment:
FWIW this should be doable without inline styles:

```tsx
const BulkActionsToast = styled(something)<{ hasMarginLeft: boolean, translateY: number }>`
  transform: ${props => `translate(-50%, ${props.translateY}%;`}
  marginLeft: ${props => props.hasMarginLeft ? NAV_SIDEBAR_WIDTH_HALF : 0};
`
```

(pardon for any mistakes in the snippet if any)

Reply:
Good point, I can remove the inline